src/views/Hero/index.view.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import PostcardStack from "components/PostcardStack/index.view";
+import EmailSubscriptionForm from "components/EmailSubscription/index.view";
+import "./Hero.scss";
+
+type HeroProps = {
+  pageName: string;
+  title: string;
+  description: string;
+};
+
+const HeroView: React.FC<HeroProps> = ({ pageName, title, description }) => {
+  return (
+    <div
+      className={cx("HeroView", { HeroView__homeView: pageName === "Home" })}
+    >
```

Comment:
Not sure if I've seen this elsewhere in the repository, but we can condense this to one line?

Reply:
In preparation for future `views`, I think we are ok leaving this in for now. When we add the rest of the views, it will look something like so: 
```tsx
className={cx(
        "HeroView",
        { HeroView__homeView: pageName === "Home" },
        { HeroView__scheduleView: pageName === "Schedule" }
      )}
```
so on and so forth. What we can do is refactor this earlier in a function and then call that function in that `className` prop.

Reply:
Ohh, that makes sense - thanks for clarifying that for me!